frontend/src/views/OrderView.vue

```diff
@@ -227,6 +227,10 @@ onMounted(async () => {
             <router-link :to="`${ROUTES.ORDER.path}/${route.params.id}/edit/offer/new`">
               <Button v-if="!(orderInfo as Order).offer" label="Angebot erstellen"></Button>
             </router-link>
+            <!-- TODO: hide button -->
```

Comment:
Since there can be more than one invoice I don't think we need to even hide the button